modules/utxo-lib/package.json

```diff
@@ -49,14 +49,15 @@
     "@bitgo/blake2b": "^3.0.3",
     "@noble/secp256k1": "1.6.3",
     "bip32": "^3.0.1",
+    "bip174": "npm:@bitgo/bip174@3.0.0",
     "bitcoin-ops": "^1.3.0",
-    "bitcoinjs-lib": "npm:@bitgo/bitcoinjs-lib@7.0.0-rc.0",
+    "bitcoinjs-lib": "git+https://github.com/BitGo/bitcoinjs-lib#3876701401d62a332c4901aa2a6f6b71c3cd799d",
```

Comment:
Is it possible to get another publish for this?

Reply:
Yes, pending a little PR merge there, we will publish.